Clarified usage for `fname_split` (#1283)
ivadomed/ivadomed: docs/source/configuration_file.rst

        "$schema": "http://json-schema.org/draft-04/schema#",
        "title": "fname_split",
        "$$description": [

>             "Name of the `joblib <https://joblib.readthedocs.io/en/latest/>`__ file that was generated during a previous training, and that contains the list of training/validation/testing filenames.\n",

Reviewer: ```suggestion
            "A `joblib <https://joblib.readthedocs.io/en/latest/>`__ file of the name split_datasets.joblib was generated during a previous training which contains the list of training/validation/testing filenames.\n",
```

Author: The way this is rephrase makes it less clear that this refers to a parameter description. 

when it starts with "Name of the `joblib..." it is clear that this is what the param is.

when it starts with "A `joblib file of the name split_datasets.joblib was generated during a previous training..." this reads like an isolated sentence, not what the param actually is
